components/meter/meter-linear.js

```diff
@@ -32,23 +49,58 @@ class MeterLinear extends LitElement  {
 				position: absolute;
 				top: 0;
 			}
+
+			.d2l-meter-linear-text {
+				color: var(--d2l-color-ferrite);
+				display: flex;
+				flex-direction: row;
+				justify-content: space-between;
+				margin: 0 6px;
+			}
+
+			.d2l-meter-linear-secondary {
+				align-self: flex-end;
+			}
+
 		`];
 	}
 
 	constructor() {
 		super();
-		this.value = 0;
 		this.max = 100;
+		this.percent = false;
+		this.value = 0;
 	}
 
 	render() {
-		const progressBarWidth = this.value / this.max * 100;
+		const percentage = this.max > 0 ? this.value / this.max * 100 : 0;
+		const primary = this.percent ? `${Math.floor(percentage)}%` : `${this.value}/${this.max}`;
+		const secondaryText = this._formatContext(this.text, this.value, this.max);
+		const secondaryTextElement = secondaryText ? html`<div class="d2l-meter-linear-secondary">${secondaryText}</div>` : html``;
 		return html `
-			<div class="d2l-meter-linear-full-bar">
-				<div class="d2l-meter-linear-inner-bar" style="width:${progressBarWidth}%;"></div>
+			<div role="img" aria-label="${secondaryText}: ${primary}; progress indicator">
```

Comment:
Why not use the [ARIA progressbar](https://www.w3.org/WAI/PF/aria/roles#progressbar) role? Seems designed for this use case...

Reply:
Ah I see your other comment now, reading...

Reply:
There is also a plan to have these animated which also may cause the `progressBar` role to also not be a great choice. At least from my understanding of what it might do.

Reply:
For the `progress-bar` it is good. For this element it will alert the user to the changes which it shouldn't do. The `progressBar` role is for progress bars not meters.

Reply:
If we did animate it initially, we'd probably keep the value static but just animate the CSS. So that'd be fine.